contracts/sources/reserve_config.move

```diff
@@ -293,6 +401,10 @@ module suilend::reserve_config {
         set_open_attributed_borrow_limit_usd(&mut builder, config.open_attributed_borrow_limit_usd);
         set_close_attributed_borrow_limit_usd(&mut builder, config.close_attributed_borrow_limit_usd);
 
+        if (bag::contains(&config.additional_fields, EModeKey {})) {
+            set_emode_ltv_for_borrow(&mut builder, *bag::borrow(&config.additional_fields, EModeKey {}))
```

Comment:
this seems wrong. `set_emode_ltv_for_borrow` is expecting `EmodeData`. but `bag::borrow(&config.additional_fields, EModeKey {})` is of type VecMap?